src/helpers/theme.js

```diff
@@ -160,6 +161,10 @@ const createAppTheme = (colorOverrides = {}) => {
         fontWeight: 500,
         align: "center",
       },
+      secondaryHighlightedRow: {
+        backgroundColor: "#cbedf2",
+      },
+      secondaryHighlightedCell: {},
```

Comment:
@sniedzielski These were actually removed in your recent PR (https://github.com/openimis/openimis-fe_js/pull/173/files) but I'm not sure if that was intentional so I brought it back to check with you

Reply:
ok, I have to double check it to ensure it's not removed by accident. If it's significant I can revert it.

Reply:
Ok. I think it's good you keep your change. At the begging I was thought  about something else regarding the removing styles. But I have to admit you are right. We can keep your change.